pkg/machines/components/vm/vmActions.jsx

```diff
@@ -71,6 +71,20 @@ const VmActions = ({ vm, config, dispatch, onStart, onInstall, onReboot, onForce
         shutdown = DropdownButtons({ buttons: buttons });
     }
 
+    let pause = null;
+    if (config.provider.canPause(state, hasInstallPhase)) {
```

Comment:
Since the operations are implemented only with the libvirt-dbus API the should be hidden for virsh only users.
Otherwise you will see the buttons but clicking them will do nothing else than spit errors in console logs.